### ISSUE TITLE
Is there any snippet to create confussion metrix inside PyABSA library?

### ISSUE THREAD

**Issue by radya13rahman** [yangheng95/PyABSA#197]
I want to access the results of the distribution of the model's predictions on the test data. So far, only f1 and accuracy are showing in the info log. Where can I access this information? furthermore is there a way to create a Confusion Matrix (CM) based on the code in the pyabse library?

**Comment by yangheng95** [comment]
You can get the logits in the apc_trainer.py, e.g.,
https://github.com/yangheng95/PyABSA/blob/99ac9963b4d8e7a87afe38ef39397df33d7bcadf/pyabsa/core/apc/training/apc_trainer.py#L393
`logits = output['logits']`

However, now api is available for CM now
